src/utils/solanaHelpers.ts

```diff
@@ -300,54 +330,62 @@ export async function getEstimateBalanceChange(connection: Connection, tx: Trans
       return prev;
     }, new Map<string, PublicKey>());
 
-    log.info(tx instanceof Transaction);
     // add selected Account incase signer is just fee payer (instruction will not track fee payer)
     accounts.set(signer, new PublicKey(signer));
-
+    const additional = tx.message?.addressTableLookups?.map((address) => address.accountKey.toBase58()) || [];
     // Simulate Transaction with Accounts
-    const result = await connection.simulateTransaction(tx.compileMessage(), undefined, Array.from(accounts.values()));
+    const addresses = Array.from(accounts.keys());
+    const combinedAddress = [...addresses, ...additional];
```

Comment:
`combinedAddress` should not be combined.
It should be `additional` or else (`additional` length is zero) `addresses`